### PR TITLE
Adding URLs and blank templates for all apps

### LINKED ISSUE #71: Adding app urls and blank templates

### PR DESCRIPTION
**Pull request by OmerCS8**
- Adding URLs to all apps.
- Adding a blank HTML template page to every feature.
- Linking URLs to templates via views
fix #71

### CODE REVIEW

**Review by MatanP12 — COMMENTED** [review]
What is the difference between signing and login?

**Comment by OmerCS8** [comment]
> What is the difference between signing and login?

Login is for registered users, and sign-in is for new users that want to register.
For now< we add users only via the admin screen, so the sign-in page isn't at top priority.
Still, I thought it would be worthwhile to prepare the ground for the moment we want to add this feature in the Feature.

**Comment by AlonShakaroff** [comment]
> > What is the difference between signing and login?
> 
> Login is for registered users, and sign-in is for new users that want to register. For now< we add users only via the admin screen, so the sign-in page isn't at top priority. Still, I thought it would be worthwhile to prepare the ground for the moment we want to add this feature in the Feature.

Sign in and login means the same thing, it's supposed to be sign up or register. @OmerCS8

File: users/urls.py
Comment (AlonShakaroff) [inline users/urls.py:7]: Sign in and login means the same thing, instead of sign in type sign up or register.

File: users/views.py
Comment (AlonShakaroff) [inline users/views.py:13]: Same here

**Review by AlonShakaroff — CHANGES_REQUESTED** [review]

File: users/views.py
Comment (MatanP12) [reply users/views.py:13]: > Login is for registered users, and sign-in is for new users that want to register.

I think we should not mess with registration, Barak told us not try implementing it, it may be complicated.
We have the 5 hardcoded users\profile in the DB, it should be enough in my opinion.

**Review by AlonShakaroff — APPROVED** [review]
LGTM

**Review by TomerNewmanPrograms — COMMENTED** [review]
Should we add views/urls to the direct_message model if its not merged it?

**Comment by OmerCS8** [comment]
> Should we add views/urls to the direct_message model if its not merged yet?

We already have a direct_message app so i think we should add the views and URLs. For now it's a blank template, but once the direct_message model will be merged we can alter the file.
(BTW we can work on the HTML and CSS of the direct messaging app even if the model hasen't been merged yet)

**Review by TomerNewmanPrograms — DISMISSED** [review]

File: ResuMe/urls.py
Comment (yulisuliman) [inline ResuMe/urls.py:24]: I think we should add a path name, something like:
 path('users/', include('users.urls'),name='Profile')

**Comment by OmerCS8** [comment]
As we agreed, I removed the sign-up URL 👍

**Review by MatanP12 — DISMISSED** [review]
LGTM

**Review by yulisuliman — APPROVED** [review]

File: posts/urls.py
Comment (natifridman) [inline posts/urls.py:8]: Although path names aren't [restricted to valid Python names](https://docs.djangoproject.com/en/4.0/topics/http/urls/#naming-url-patterns-1), I still think it should be lowercase

File: users/urls.py
Comment (natifridman) [inline users/urls.py:6]: Although path names aren't [restricted to valid Python names](https://docs.djangoproject.com/en/4.0/topics/http/urls/#naming-url-patterns-1), I still think it should be lowercase

File: direct_message/urls.py
Comment (natifridman) [inline direct_message/urls.py:5]: Although path names aren't [restricted to valid Python names](https://docs.djangoproject.com/en/4.0/topics/http/urls/#naming-url-patterns-1), I still think it should be lowercase

File: ResuMe/urls.py
Comment (natifridman) [inline ResuMe/urls.py:26]: Do we want to add `/users/` for every user URL? Same with `/direct`

File: posts/urls.py
Comment (OmerCS8) [reply posts/urls.py]: fixed

File: ResuMe/urls.py
Comment (OmerCS8) [reply ResuMe/urls.py:26]: I think that the URLs are well organized that way.
So if I want to search for a user's profile, I'll find it under `/users/`.
Regarding `direct`, I think it is convenient as well, and I saw that they use a similar URL on Instagram.
image

**Review by MatanP12 — COMMENTED** [review]
LGTM

**Review by yulisuliman — APPROVED** [review]

**Review by TomerNewmanPrograms — APPROVED** [review]
LGTM

**Review by AlonShakaroff — DISMISSED** [review]
LGTM

**Review by natifridman — APPROVED** [review]

**Review by yulisuliman — DISMISSED** [review]

**Comment by yulisuliman** [comment]
@OmerCS8 Please squash all the commits

**Review by TomerNewmanPrograms — APPROVED** [review]

**Comment by OmerCS8** [comment]
> @OmerCS8 Please squash all the commits

Squashed all the commits and ready to merge 👍 . @natifridman @halleelyasaf @abiton1

**Review by TomerNewmanPrograms — APPROVED** [review]
LGTM

**Comment by natifridman** [comment]
> > @OmerCS8 Please squash all the commits
> 
> Squashed all the commits and ready to merge 👍 . @natifridman @halleelyasaf @abiton1

@OmerCS8 Great! we need one more approval

**Review by MatanP12 — APPROVED** [review]
LGTM

**Review by AlonShakaroff — APPROVED** [review]
LGTM

**Review by natifridman — APPROVED** [review]

**Pull request merged**
